### PR TITLE
Account payment, account balance and account echange endpoints (testing to be done)

### PR DESCRIPTION
**Pull request by gip**

### CODE REVIEW

File: lib/hbase/hbase-client.js [hunk 1]
```diff
@@ -244,14 +244,33 @@ HbaseClient.prototype.getAccountPayments = function (options, callback) {
...
+  var filterString= this.buildSingleColumnValueFilters(maybeFilters);
+
+  this.getScanWithMarker(this, {
+    table        : table,
+    startRow     : startRow,
+    stopRow      : endRow,
+    limit        : options.limit,
+    descending   : options.descending,
+    marker       : options.marker,
+    type         : options.type,
```
Comment (shekenahglory) [inline lib/hbase/hbase-client.js]: Type shouldn't be here right?

File: lib/hbase/hbase-client.js [hunk 2]
```diff
@@ -244,14 +244,33 @@ HbaseClient.prototype.getAccountPayments = function (options, callback) {
...
+    options.currency= options.currency.toUpperCase();
+  }
+
+  if(options.type) {
+    options.type= options.type.toLowerCase();
+  }
+
+  var maybeFilters =
+  [{ qualifier: 'currency', value: options.currency, family: 'f', comparator: '=' },
+   { qualifier: options.type, value: options.account, family: 'f', comparator: '=' }];
```
Comment (shekenahglory) [inline lib/hbase/hbase-client.js]: type should be 'sent' or 'received' with logic here to translate it into the appropriate filters for 'source' and 'destination' columns

File: lib/hbase/hbase-client.js (hunk 1 above)
Comment (gip) [inline lib/hbase/hbase-client.js]: Rugit, type is used when building the filter string and isn't used in getScan directly. Should be removed indeed.

File: lib/hbase/hbase-client.js [hunk 3]
```diff
@@ -272,38 +291,24 @@ HbaseClient.prototype.getAccountPayments = function (options, callback) {
...
   var keyBase = options.account;
-  var table = 'lu_account_balance_changes';
-  var startRow;
-  var endRow;
+  var table = 'account_balance_changes';
+  var startRow = options.account + '|' + utils.formatTime(options.start);
+  var endRow   = options.account + '|' + utils.formatTime(options.end);
 
-  if (!options.currency) callback({error:"must provide a currency", code:400});
+  if (!options.currency) { callback({error:"must provide a currency", code:400}); }
```
Comment (shekenahglory) [inline lib/hbase/hbase-client.js]: currency is no longer required here.  We should have filters for currency and issuer

File: lib/hbase/hbase-client.js (hunk 2 above)
Comment (gip) [inline lib/hbase/hbase-client.js]: Done. Supporting both sent/received and source/destination.

**Comment by shekenahglory** [comment]
LGTM

**Pull request merged**